helm/prometheus-meta-operator/templates/recording-rules/cortex.rules.yml

```diff
@@ -10,184 +10,180 @@ spec:
+      record: aggregation:kubernetes:deployment_replicas_desired
+    - expr: count(kube_deployment_created) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:deployment_total
+    - expr: count(kube_endpoint_created) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:endpoint_total
+    - expr: count(kube_ingress_created) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:ingress_total
+    - expr: sum(kube_job_status_active) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:job_active
+    - expr: sum(kube_job_complete) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:job_complete
+    - expr: count(kube_job_created) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:job_total
+    - expr: min((kube_node_created) and on (node) (label_replace(kube_pod_created{instance=~".*master.*", pod=~"k8s-api-server.*"}, "node", "$1", "pod", "k8s-api-server-(.*)"))) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:master_node_created
+    - expr: count(kube_namespace_created) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:namespace_total
+    - expr: sum(kube_node_status_allocatable_cpu_cores) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_allocatable_cpu_cores_total
+    - expr: sum(kube_node_status_allocatable_memory_bytes) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_allocatable_memory_bytes
+    - expr: sum(kube_node_status_allocatable_pods) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_allocatable_pods_total
+    - expr: sum(kube_node_status_condition{condition="DiskPressure", status="true"}) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_disk_pressure_total
+    - expr: sum(kube_node_status_condition{condition="MemoryPressure", status="true"}) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_memory_pressure_total
+    - expr: sum(kube_node_status_condition{condition="Ready", status="false"}) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_not_ready_total
+    - expr: sum(kube_node_status_condition{condition="PIDPressure", status="true"}) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:node_pid_pressure_total
+    - expr: sum(kube_node_role) by (cluster_type, role, cluster_id)
+      record: aggregation:kubernetes:node_total
+    - expr: sum(kube_persistentvolume_capacity_bytes) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:persistentvolume_capacity_bytes
+    - expr: count(kube_persistentvolume_info) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:persistentvolume_total
+    - expr: sum(kube_persistentvolumeclaim_resource_requests_storage_bytes) by (cluster_type, cluster_id)
+      record: aggregation:kubernetes:persistentvolumeclaim_resource_requests_storage_bytes
```

Comment:
I don't see this being useful, but it's anyway impossible to tell here whats used and whats not.

Reply:
I am thinking that once we have all the cortex dashboard as code, we can check which aggregations we use or not

Reply:
Let's do that then